config/routes.rb

```diff
@@ -15,8 +15,8 @@
 
   resources :submissions
 
-  resources :users
+  resource :profile, only: [:edit, :update]
```

Comment:
You could have changed this from `resources :users` to `resource :profile`, like you did, and then you could have left the `devise_for :users` in place.  To me, I like both changes because registration, sign_up, and sign_out, etc. are really done within the scope of a single user instead of plural `users`.